### PR TITLE
Ssp workaround

### PR DESCRIPTION
**Pull request by manorman5**
hey! we only projected delta method output for SSP3, but we projected point estimate output for SSP2 and SSP4. Thus, when I ran the quantiles extraction code for a config which requested information from both point estimate and delta method output I would get a keyerror. I implemented a fix to avoid the keyerror. I don't feel like my fix was particularly hacky, but correct me if I am wrong 🙃 . Also very open to feedback on if I should be including more or different info in this request.

### CODE REVIEW

File: gcp/extract/quantiles.py [hunk 1]
```diff
@@ -42,6 +42,14 @@
 
 for filestuff in data:
     print "Creating file: " + str(filestuff)
+    
+    if configs.is_parallel_deltamethod(config):
+        try:
+            output_check = parallel_deltamethod_data[filestuff].keys()
+        except KeyError:
+            print str(filestuff) + " is not in delta method output. Skipping model specification..."
+            continue
```
Comment (jrising) [inline gcp/extract/quantiles.py]: Can we replace this with:
```
if configs.is_parallel_deltamethod(config) and filestuff not in parallel_deltamethod_data:
    print str(filestuff) + " is not in delta method output. Skipping model specification..."
    continue
```
I think that is somewhat more legible.

**Comment by jrising** [comment]
Can you test my solution, if let me know if you disagree with the change?

File: gcp/extract/quantiles.py (hunk 1 above)
Comment (manorman5) [reply gcp/extract/quantiles.py]: yep, much prettier :). ill test it out now.

**Comment by jrising** [comment]
@brews I removed you as a reviewer. I think just one of us needs to review.

**Comment by jrising** [comment]
Great. Looks fine to me.

**Pull request merged**